tests/scripts/pulp_rpm/test_rpm_sync_publish.sh

```diff
@@ -6,9 +6,9 @@
 pulp debug has-plugin --name "rpm" || exit 3
 
 cleanup() {
-  pulp rpm remote destroy --name "cli_test_rpm_remote" || true
-  pulp rpm repository destroy --name "cli_test_rpm_repository" || true
   pulp rpm distribution destroy --name "cli_test_rpm_distro" || true
+  pulp rpm repository destroy --name "cli_test_rpm_repository" || true
+  pulp rpm remote destroy --name "cli_test_rpm_remote" || true
```

Comment:
Cleanup in reverse order?
Fine for me! :+1:

Reply:
Just making the cleanup() match the order at test-end :)

Reply:
> Is this a rather recently added feature in pulp_rpm?
> If yes, do you think we can guard it's use by `has_plugin`?
> 
> edit: Probybly `needs_plugin` being lazy is the better choice.

The feature has been in pulp_rpm since June , released in pulp_rpm 3.5 - does it really need a guard?

Reply:
June is older than any CLI-Release. I believe we agreed to support about 5 releases back. So assuming your test would fail on earlier versions of pulp_rpm (which it doesn't in the cli) i think we are safe.